sample/library/build.gradle

```diff
@@ -23,7 +23,12 @@ android {
 
 dependencies {
   implementation deps.release.runtime
-  annotationProcessor deps.release.compiler
+  implementation project(':sample:base-library')
+
+  //TODO this change is just to demonstrate that the new compiler works well
+  //but should be reversed when the new version is released
+  //annotationProcessor deps.release.compiler
+  annotationProcessor project(':butterknife-compiler')
```

Comment:
I can revert that as soon as you want, I just use it for debugging purposes. I will probably open a ticket soon on the BK repo: in another project we now use composite builds to manage samples and it works great. It allows to have a clean sample and also use the current modules being developed for testing purposes.